mrp_stock_lot_analytic/models/mrp_production.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class MrpProduction(models.Model):
+    _inherit = "mrp.production"
+
+    def _prepare_stock_lot_values(self):
+        self.ensure_one()
```

Comment:
```suggestion
```
A minor suggestion. We can omit this when we make a super() call the first thing in the method.